### ISSUE TITLE
How to Choose Loss Functions When Training Deep Learning Neural Networks

### ISSUE THREAD

**Issue by jatinkumar762** [jatinkumar762/AI_Articles#58]
### Article Link
[Link](https://machinelearningmastery.com/how-to-choose-loss-functions-when-training-deep-learning-neural-networks/)
